configs/scripts/program/dump.sh

```
@@ -15,12 +17,12 @@ cd $(dirname $(dirname $(dirname $SCRIPT_DIR)))
 
 OUTPUT=$1
 
-if [ -z ${RPC+x} ]; then
-    RPC="https://api.mainnet-beta.solana.com"
-fi
+RPC_MAINNET="https://api.mainnet-beta.solana.com"
+RPC_DEVNET="https://api.devnet.solana.com"
 
 if [ -z "$OUTPUT" ]; then
     echo "missing output directory"
+    cd ${CURRENT_DIR}
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)